synthtool/languages/java.py

```diff
@@ -306,6 +309,7 @@ def bazel_library(
         suffix="-java",
         destination_name=destination_name,
         cloud_api=cloud_api,
+        diregapic=kwargs.get("diregapic", False),
```

Comment:
We can add to the named param list of `gapic_library()` and `bazel_library()` to avoid the `kwargs.get()`